stat.go

```diff
@@ -3,15 +3,70 @@ package procfs
 import (
 	"bufio"
 	"fmt"
+	"io"
 	"os"
 	"strconv"
 	"strings"
 )
 
+const (
+	userHz = 100
```

Comment:
I guess we should add a comment how we came up with this constant and why we consider it safe to use a constant. There should be something in the node_exporter we could just use here.